### DISCUSSION TITLE
VRP Compactness

### DISCUSSION THREAD

**Discussion by rahulj2409** [google/or-tools#2700] [Routing (and legacy CP) questions]
Is there a way to measure "compactness" of a route and minimize it?

**Comment by lperron** [comment]
Span ?

**Reply by rahulj2409** [reply]
Would this be related to making the routes look "tighter" in a visual sense? /IE related to standard deviation kinda stuff?

Thanks!

**Reply by lperron** [reply]
the span is the sum of transit with the initial waiting time at the depot. This was asked multiple times here and on the mailing list.
